### PR TITLE
next release v2.6.0

### PR DESCRIPTION
**Pull request by casperdcl**
- add `acr` module
- add `img.signa` module
- misc `prc` & `img` updates
- fix/migrate `pkg_resources` => `importlib_resources`
- misc code tidy
- misc build/CI/framework/test updates
  + fix CI build & test workflow
  + add Py3.11, drop Py3.6 support
  + migrate `setup.cfg` => `pyproject.toml`
  + update pre-commit hooks

### CODE REVIEW

**Comment by pjmark** [comment]
from me it is all good

**Comment by casperdcl** [comment]
Pushed lots of fixes, but left some errors (see https://github.com/NiftyPET/NIMPA/actions/runs/5193633461/jobs/9364422220) because I'm unsure about them

File: niftypet/nimpa/prc/imio.py [hunk 1]
```diff
@@ -1053,21 +1068,21 @@ def dcm2im(fpth):
     fpth can be a list of DICOM files or a path (string) to the folder with DICOM files.
     '''
     # possible DICOM file extensions
-    ext = ('dcm', 'DCM', 'ima', 'IMA')
+    ext = 'dcm', 'ima'
```
Comment (casperdcl) [inline niftypet/nimpa/prc/imio.py:1071]: btw can this use `dcmext` (`'dcm', 'ima', 'img'`) instead?

File: niftypet/nimpa/acr/proc.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    imio.create_dir(outdir)
+
+    if isinstance(indat, (str, PurePath)) and Path(indat).is_dir():
+        # CONVERT TO NIfTI
+        if not imio.dcmdir(indat):
+            raise IOError('the provided folder does not contain DICOM files')
+        for f in chain(outdir.glob('*.nii*'), outdir.glob('*.json')):
+            # remove previous files
+            os.remove(f)
+        run([dcm2niix.bin, '-i', 'y', '-v', 'n', '-o', outdir, '-f', '%f_%s', str(indat)])
```
Comment (casperdcl) [inline niftypet/nimpa/acr/proc.py:36]: this was `'f', '%f_%s'` instead of `'-f', '%f_%s'`. Presumably latter is correct.

**Pull request merged**